spring-cloud-cloudfoundry-commons/src/test/java/org/springframework/cloud/cloudfoundry/CloudFoundryClientAutoConfigurationTest.java

```diff
@@ -58,7 +57,7 @@ private static String envVarFromProperty(String propertyName) {
 	}
 
 	@Test
-	@Ignore // see https://github.com/spring-cloud/spring-cloud-cloudfoundry/issues/65
+	// @Ignore // see https://github.com/spring-cloud/spring-cloud-cloudfoundry/issues/65
```

Comment:
Can you remove this line? There's no reason to keep this.

Reply:
Done